.idea/modules.xml

```diff
@@ -2,7 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/flowcrypt-android.iml" filepath="$PROJECT_DIR$/flowcrypt-android.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/modules/flowcrypt-android.iml" filepath="$PROJECT_DIR$/.idea/modules/flowcrypt-android.iml" />
```

Comment:
@IvanPizhenko Please don't save such changes. Android Studio and IDEA have different folders for settings. Just skip them.

Reply:
@DenBond7 I'm using latest Android Studio for working in this repo. so not sure why that has happened.